protocol/src/main/java/io/zeebe/protocol/intent/ErrorIntent.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.protocol.intent;
+
+public enum ErrorIntent implements Intent {
+  UNEXPECTED_EXCEPTION((short) 0);
```

Comment:
This intent doesn't fit well to out other intents. Maybe we can rename it to `CREATED`?

Reply:
Hm or `ERROR`?

Reply:
ok